apps/test/unit/util/curriculumRecommenderTestCurricula.js

```diff
@@ -292,6 +296,7 @@ const FULL_TEST_COURSES = [
   },
   {
     key: 'fullTestCourse5',
+    course_offering_id: 5,
```

Comment:
Are these already present on the objects used by the components? (ie outside of the tests)

Reply:
Oh thanks for catching this! I had misunderstood the field names since this is how it's listed in `section.rb`, but it should just be `id`, I'll update these files accordingly!

Reply:
I'll also re-test everything to make sure it works as expected!